test/unit/Escrow.t.sol

```diff
@@ -736,15 +736,13 @@ contract EscrowUnitTests is UnitTest {
         assertEq(signallingEscrowDetails.totalUnstETHFinalizedETH.toUint256(), 0);
     }
 
-    function test_markUnstETHFinalized_RevertOn_UnexpectedEscrowState() external {
+    function test_markUnstETHFinalized_RevertOn_EmptyUnstETHIds() external {
```

Comment:
`test_markUnstETHFinalized_RevertOn_UnexpectedEscrowState` should be preserved but modified to pass a non-empty list of unstETHIds